Header header

#RC data

uint16[8]   rc_channel

#Control data

uint16[6]   control_axes
int32[]     control_buttons